iconCompiler/reactIcon.js

```diff
@@ -0,0 +1,27 @@
+const { createComponentName, convertPxToRem } = require('./tools')
+
+const fileHeader = `/*
+DON'T EDIT THIS FILE
+It was auto-generated by "iconCompiler" script
+*/
```

Comment:
generated file should be in the .gitignore ?

Reply:
as it is part of the package, for now we keep it here, waiting for a CI solution.